apollo-api/src/commonMain/kotlin/com/apollographql/apollo3/api/ApolloResponse.kt

```diff
@@ -8,15 +8,15 @@ import kotlin.jvm.JvmName
 /**
  * Represents a GraphQL response. GraphQL responses can be be partial responses so it is valid to have both data != null and errors
  */
-class ApolloResponse<out D : Operation.Data>(
+class ApolloResponse<D : Operation.Data> @Deprecated("Please use ApolloRequest.Builder methods instead. This will be removed in v3.0.0.") constructor(
```

Comment:
```suggestion
class ApolloResponse<D : Operation.Data> @Deprecated("Please use ApolloResponse.Builder methods instead. This will be removed in v3.0.0.") constructor(
```

Reply:
Good catch 🙏